d2bs/kolbot/libs/common/Prototypes.js

```diff
@@ -1104,3 +1104,149 @@ if (!Array.prototype.find) {
 		writable: true
 	});
 }
+
+/**
+ * @description very simple but handy function, that either returns the first element, or undefined
+ */
+if (!Array.prototype.first) {
+	Array.prototype.first = function () {
+		return this.length > 0 ? this[0] : undefined;
+	};
+}
+
+/**
+ * Just a simple wrapper around unit.getItem(), to avoid the use of .getNext() but just get an array,
+ * and the guaranty of getting an array ack if the unit doesnt have items
+ * @param args
+ * @returns Unit[]
+ */
+Unit.prototype.getItems = function (...args) {
+	let item = this.getItem.apply(this, args), items = [];
+	item && items.push(copyUnit(item));
+
+	while (item && item.getNext()) {
+		items.push(copyUnit(item));
+	}
+
+	return items;
+};
+
+/**
+ * @description ArachnidMesh.useChargedSkill() casts a venom or me.useChargedSkill(278);
+ * @param {int} skillId = undefined
+ * @param {int} x = undefined
+ * @param {int} y = undefined
+ */
+Unit.prototype.useChargedSkill = function (...args) {
```

Comment:
change name to cast to correspond with skill function?

Reply:
Any suggestions?

Reply:
add @returns?

Reply:
- changed name to useChargedSkill.
- Added @return
- Added @throws